TAF/testScenarios/integrationTest/UC_ACL/acl.robot

```diff
@@ -1,15 +1,62 @@
 *** Settings ***
-Force Tags    Skipped
+Resource  TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource  TAF/testCaseModules/keywords/core-metadata/coreMetadataAPI.robot
+Suite Setup  Run Keywords  Skip If  $SECURITY_SERVICE_NEEDED != 'true'
+             ...        AND  Setup Suite
+Suite Teardown  Run Teardown Keywords
 
 *** Variables ***
 ${SUITE}          Proxy Auth - ACL
 ${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/proxy-auth-acl.log
+${proxyAuthUrl}   ${URI_SCHEME}://${BASE_URL}:${PROXY_AUTH_PORT}
+${keyUri}  /key
 
 *** Test Cases ***
 ACL001 - Send query API by external JWT token
     Given Generate External JWT Token
-    And Generate Validation Key
-    And Get Proxy Token Using api-gateway-token Script
-    And Add Validation Key
-    When Query All Device Profiles By External JWT Token
-    Then Device Profiles Should Be Returned
+    And Get Token
+    And Set Proxy And JWT Tokens  ${jwt_token}  ${external_token}
+    And Create Key  ${key_body}
```

Comment:
It is not necessary to add a new keyword `Set Proxy And JWT Tokens`. Just use `${jwt_token} `in Create Key, and then set `${external_token}` to `${jwt_token}`.